documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -799,11 +800,14 @@ EOF
 <5> Specify the name of the `NetworkMap` CR.
 <6> Specify a storage mapping even if the VMs to be migrated are not assigned with disk images. The mapping can be empty in this case.
 <7> Specify the name of the `StorageMap` CR.
-<8> You can use either the `id` _or_ the `name` parameter to specify the source VMs. +
-<9> Specify the VMware vSphere VM moRef. To retrieve the moRef, see xref:retrieving-vmware-moref_mtv[Retrieving a VMware vSphere moRef].
-<10> Optional: You can specify up to two hooks for a VM. Each hook must run during a separate migration step.
-<11> Specify the name of the `Hook` CR.
-<12> Allowed values are `PreHook`, before the migration plan starts, or `PostHook`, after the migration is complete.
+<8> By default, virtual network interface controllers (vNICs) change during the migration process. This results in vNICs that are set with a static IP in vSphere losing their IP. To avoid this by preserving static IPs of VMs, set `preserveStaticIPs` to `true`. {project-short} issues a warning message about any VMs with a Windows operating system for which vNIC properties are missing. To retrieve any missing vNIC properties, run those VMs in vSphere in order for the vNIC properties to be reported to {project-short}.
+<9> You can use either the `id` _or_ the `name` parameter to specify the source VMs.
```

Comment:
@anarnold97 Callout 8 is longer than our usual laconic callouts. It's based on the comments from the UI section, trimmed somewhat, but still longer than the rest. Please advise if you see any fluff. Ideally, I'd insert the snippet after callout 8, but bccutil did not respond well to my attempts at that.